delphi-checks/src/test/java/au/com/integradev/delphi/checks/FormDfmCheckTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Sonar Delphi Plugin
+ * Copyright (C) 2024 Integrated Application Development
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package au.com.integradev.delphi.checks;
+
+import au.com.integradev.delphi.builders.DelphiTestUnitBuilder;
+import au.com.integradev.delphi.checks.verifier.CheckVerifier;
+import org.junit.jupiter.api.Test;
+
+class FormDfmCheckTest {
+  @Test
+  void testNormalClassWithoutDfmShouldNotAddIssue() {
+    CheckVerifier.newVerifier()
+        .withCheck(new FormDfmCheck())
+        .withSearchPathUnit(createVclForms())
+        .onFile(
+            new DelphiTestUnitBuilder()
+                .appendDecl("type")
+                .appendDecl("  TFoo = class(TForm)")
```

Comment:
For clarity, might be a good idea to change the parent class `TForm` here to something else,
e.g. `TObject`. (This test only passes because it can't resolve the type.)

Reply:
Whoops, I didn't mean to inherit from an unresolved type here. Good catch.